pages/docs/index.js

```diff
@@ -277,6 +277,16 @@ export default function DocsIndex() {
       title: "Extra",
       description: "More information",
       pages: [
+        {
+          name: "Profile Tips",
+          path: "/docs/profile-tips",
+          description:
+          "Tips on making your LinkFree profile stand out.",
+          category: { 
+            name: "Intermediate",
```

Comment:
Creating a Profile is the most basic step in LinkFree and therefore this should be for `beginners`:
```suggestion
            name: "Beginner",
```